src/components/FeedInterstitials.tsx

```diff
@@ -176,9 +173,12 @@ function useExperimentalSuggestedUsersQuery() {
 }
 
 export function SuggestedFollows({feed}: {feed: FeedDescriptor}) {
+  const {currentAccount} = useSession()
   const [feedType, feedUri] = feed.split('|')
   if (feedType === 'author') {
-    return <SuggestedFollowsProfile did={feedUri} />
+    return currentAccount?.did !== feedUri ? (
```

Comment:
nit: this would be a bit easier to read without negation — just following nested structure

```
if (feedType === 'author') {
  if (currentAccount?.did === feedUri) {
    ..
  } else {
    ..
} else {
  ..
}
```